elastic/kibana
# [Security Solution] Template unit tests

## Summary

1. Adding more unit tests for import timelines' api.
2. Fix known issue: Unexpected errors occurs when creating template timeline via import without templateTimelineId / templateTimelineVersion. - https://github.com/elastic/kibana/issues/69972

To verify this PR:
1. Download  [timelines_export.txt](https://github.com/elastic/kibana/files/4953375/timelines_export.txt)

2. Change `.txt` into `.ndjson`
3. Import the file and see if it's created successfully as a custom template.

### Checklist

Delete any items that are not applicable to this PR.


### For maintainers
